netty/src/test/java/io/grpc/netty/NettyServerBuilderTest.java

```diff
@@ -69,4 +69,52 @@ public void failIfKeepAliveTimeoutNegative() {
 
     NettyServerBuilder.forPort(8080).keepAliveTimeout(-10L, TimeUnit.HOURS);
   }
+
+  @Test
+  public void failIfMaxConcurrentCallsPerConnectionNegative() {
+    thrown.expect(IllegalArgumentException.class);
+    thrown.expectMessage("max must be positive");
+
+    NettyServerBuilder.forPort(8080).maxConcurrentCallsPerConnection(0);
```

Comment:
nit: Declare `builder = NettyServerBuilder.forPort(8080)` prior to `thrown.expect`

Reply:
Actually... we should probably just make that a field at the top, since so many of these tests will use that. That said, these new tests are working like the old tests, so it seems fair.